seatunnel-engine/seatunnel-engine-common/src/main/resources/hazelcast-client.yaml

```diff
@@ -21,3 +21,17 @@ hazelcast-client:
   network:
     cluster-members:
       - localhost:5801
+      - localhost:5802
+      - localhost:5803
+      - localhost:5804
+      - localhost:5805
+      - localhost:5806
+      - localhost:5807
+      - localhost:5808
+      - localhost:5809
+      - localhost:5810
+      - localhost:5811
+      - localhost:5812
+      - localhost:5813
+      - localhost:5814
+      - localhost:5815
```

Comment:
Need to submit this address list?

Reply:
> Need to submit this address list?

Yes, The client need try this address list and get the right one which `clustername` is same as client.